ynput/OpenPype
# Hiero: Effect Categories - OP-7397

## Changelog Description
This PR introduces `Effect Categories` for the Hiero settings. This allows studios to split effect stacks into meaningful subsets.

## Testing notes:
1. Setup `Effect Categories` in Hiero addon settings (see documentation).
2. Setup a clip in Hiero with effects corresponding to the settings.
3. Publish clip and effects.
4. Load into Nuke and verify the effects are grouped correctly.
